components/sections/matrix/content.ja-JP.md

```diff
@@ -0,0 +1,120 @@
+# 行列演算としてのベジエ曲線の曲率
+
+ベジエ曲線は、行列演算の形でも表現することができます。ベジエ曲線の式を多項式基底と係数行列で表し、実際の座標も行列として表現するのです。これがどういうことを意味しているのか、3次ベジエ曲線について見てみましょう。
+
+\[
+B(t) = P_1 \cdot (1-t)^3 + P_2 \cdot 3 \cdot (1-t)^2 \cdot t + P_3 \cdot 3 \cdot (1-t) \cdot t^2 + P_4 \cdot t^3
+\]
+
+実際の座標を一旦無視すると、次のようになります。
+
+\[
+B(t) = (1-t)^3 + 3 \cdot (1-t)^2 \cdot t + 3 \cdot (1-t) \cdot t^2 + t^3
+\]
+
+これは、4つの項の和になっています。
+
+\[
+  \begin{matrix}
+   ... & = & (1-t)^3 \\
```

Comment:
As I'm unfamiliar with technical writing in Japanese involving maths, is the "..." convention understood well enough, or are other symbols typically used in Japanese texts for this placeholder?